reflex-dom-core/ChangeLog.md

```diff
@@ -40,6 +40,9 @@
 
  * Add `< 0.7` upper bound for reflex
 
+* *breaking change* The wheel event now is of `WheelEventResult` rather than `()`.
+  This provides information about the wheels motion beyond the fact that it merely moved.
```

Comment:
```suggestion
* *breaking change* The wheel event now is a `WheelEventResult` rather than `()`.
  This provides information about the wheel's motion beyond the fact that it merely moved.
```